Lombiq.HelpfulLibraries.Cli/Lombiq.HelpfulLibraries.Cli.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Have we documented somewhere that we want and should be using this option? Seems like a big change to me with regards to our other projects.

Reply:
I think it must be super safe if Microsoft has added it to their default project templates. Anyway, I have created a poll [in Teams](https://teams.microsoft.com/l/message/19:525a1a3656c845ae92c5e2c0d3617cc7@thread.skype/1653392649090?tenantId=959ede5a-38e9-44e6-adc8-85553beb8220&groupId=d6a5d775-6fb9-41dc-a7bb-481d4f884046&parentMessageId=1653392649090&teamName=Lombiq%20Developers&channelName=General&createdTime=1653392649090) because I couldn't find any internal docs on it.

Reply:
I have removed it for now, to expedite the issue resolution.

Reply:
I thought that was not necessary after [Zoltán's comment](https://teams.microsoft.com/l/message/19:525a1a3656c845ae92c5e2c0d3617cc7@thread.skype/1653394111309?tenantId=959ede5a-38e9-44e6-adc8-85553beb8220&groupId=d6a5d775-6fb9-41dc-a7bb-481d4f884046&parentMessageId=1653392629945&teamName=Lombiq%20Developers&channelName=General&createdTime=1653394111309) in Teams:
> For OSOE-49 I'd leave it enabled
🤔

Reply:
I've missed that. Meh, it's no big deal either way.